app/models/solid_queue/semaphore.rb

```diff
@@ -41,16 +41,30 @@ def signal
       end
 
       private
+
```

Comment:
In this project I'm following the style of not having a space after `private` when the class has public and private methods, so this one was on purpose 😅